src/hooks/useSearchResults.ts

```diff
@@ -24,9 +27,19 @@ export const useSearchResults = () => {
 
   const searchResults = useCallback(
     (currentQuery: string) => {
-      const currentResults = storeIndex
-        .search(currentQuery, { expand: true })
-        .map(({ ref }) => storeIndex.documentStore.getDoc(ref) as SearchResult);
+      const localeResults: SearchResult[] = [];
```

Comment:
Nit:

```tsx
const localeResults: SearchResult[] = [];
const fallbackResults: SearchResult[] = [];

storeIndex.search(currentQuery, { expand: true }).forEach(({ ref }) => {
  const result = storeIndex.documentStore.getDoc(ref) as SearchResult;

  [result.locale === locale ? localeResults : fallbackResults]
    ['push'](result)
});
```

Reply:
@ovflowd 

Since we only want to keep result item in locale "en" as fallback result, the check `result.locale === 'en'` is needed to me so we don't want to add item that is not in locale "en" and also not in current locale :D

Reply:
I see, at least if you could add some spacing, between line 31-32 and line 33-34 that'd be great xD